test/tablet.py

```diff
@@ -128,14 +132,16 @@ def __str__(self):
     return 'tablet: uid: %d web: http://localhost:%d/ rpc port: %d' % (
         self.tablet_uid, self.port, self.grpc_port)
 
-  def mysqlctl(self, cmd, extra_my_cnf=None, with_ports=False, verbose=False):
+  def mysqlctl(self, cmd, extra_my_cnf=None, with_ports=False, verbose=False,
```

Comment:
Hm... I was assuming that github doesn't wrap the line until it's more than 80 characters long. Oh well... :(